stdlib/ballerina-websub/src/main/ballerina/websub.hub/hub_configuration.bal

```diff
@@ -54,7 +54,7 @@ http:SecureSocket? secureSocket = getSecureSocketConfig();
 
 documentation {
     Function to retrieve the URL for the Ballerina WebSub Hub, to which potential subscribers need to send
-    subscription/unsubscription requests.
+    subscription/unsubscription requests
```

Comment:
Do we have to remove the period at the end? Is this a convention we have adopted?

Reply:
Removed for consistency, but it seems like there is no convention. Discussed offline with Nirmal, and it looks like it would be better to include periods at the end of function, object, etc. descriptions, but not at the end of field, parameter. etc. descriptions which would be listed in a table.

We may have to come up with a convention?

Reply:
Better to add period to the last sentence of doc description. It is just a normal sentence.

Reply:
Yes, will update.

Reply:
Fixed with 877f5b9